src/usb.rs

```diff
@@ -28,6 +28,7 @@ unsafe impl UsbPeripheral for Peripheral {
 
     fn enable() {
         let rcc = unsafe { (&*RCC::ptr()) };
```

Comment:
Newbie question, @therealprof: can this reference to the pointer of RCC `unsafe` be avoided somehow? I'm curious.

Reply:
Only if you require passing in `RCC` into your function or putting the method on the "owner" of RCC.